engine/swarm/swarm_manager_locking.md

```diff
@@ -0,0 +1,153 @@
+---
+description: Automatically lock Swarm managers to protect encryption keys
+keywords:
+- swarm, manager, lock, unlock, autolock, encryption
+title: Lock your swarm to protect its encryption key
+---
+
+In Docker 1.13 and higher, the RAFT logs used by swarm managers are encrypted on
+disk by default. This at-rest encryption protects your service's configuration
+and data from attackers who gain access to the encrypted RAFT logs. One of the
+reasons this feature was introduced was in support of the new [Docker
+secrets](secrets.md) feature.
+
+When Docker restarts, the encryption key needed to encrypt and decrypt RAFT logs
+is loaded into each manager node's running memory from disk. Docker 1.13
+introduces the ability to protect the encryption key at rest by _autolocking_
+your swarm.
+
+When Docker restarts or a new manager attempts to join the swarm, or whenever
```

Comment:
When a new manager joins, the unlock key does not need to be provided, because that new manager does not have any data on disk that needs to be loaded.  The new manager will obtain the the unlock key from the raft leader via propagation of the rest of the raft data.